src/en/darthsdroids/src/eu/kanade/tachiyomi/extension/en/darthsdroids/DarthsDroids.kt

```diff
@@ -0,0 +1,222 @@
+            // a `.html` page, but a directory path with an `index.html`. So
+            // we’re returning `solo/$pageUrl`.
+            maybeRelativeTo + pageUrl
+        }
+
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        var archiveResponse = client.newCall(GET(baseUrl + manga.url, headers)).execute()
+        if (!archiveResponse.isSuccessful) {
+            archiveResponse = client.newCall(GET(mainArchiveUrl, headers)).execute()
+        }
+
+        // Fingers crossed they don’t come up with yet another different archive structure.
+        // Technically `manga.url` is »incorrect« for the main archive, but that parameter
+        // is ignored for almost all archives anyway. Only those not pointing at a `.html`
+        // page matter.
+        return Observable.just(fetchChaptersForDatedAndDatelessArchives(archiveResponse.asJsoup(), manga.url))
+    }
+
+    override fun fetchPageList(chapter: SChapter): Observable<List<Page>> =
+        Observable.just(listOf(Page(0, chapter.url)))
+
+    override fun fetchImageUrl(page: Page): Observable<String> {
+        val comicPage = client.newCall(GET(baseUrl + page.url, headers)).execute().asJsoup()
+        // Careful. For almost all images it’s `div.center>p>img`, except for pages released on
+        // April’s Fools day, when it’s `div.center>p>a>img`.
+        val imageUrl = comicPage.selectFirst("div.center img")!!.attr("src")
+        return Observable.just(baseUrl + imageUrl)
+    }
+
+    override fun popularMangaRequest(page: Int): Request = throw UnsupportedOperationException()
+
+    override fun popularMangaParse(response: Response): MangasPage = throw UnsupportedOperationException()
+
+    override fun mangaDetailsParse(response: Response): SManga = throw UnsupportedOperationException()
+
+    override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> = throw UnsupportedOperationException()
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request = throw UnsupportedOperationException()
+
+    override fun searchMangaParse(response: Response): MangasPage = throw UnsupportedOperationException()
+
+    override fun latestUpdatesRequest(page: Int): Request = throw UnsupportedOperationException()
+
+    override fun latestUpdatesParse(response: Response): MangasPage = throw UnsupportedOperationException()
+
+    override fun chapterListRequest(manga: SManga): Request = throw UnsupportedOperationException()
+
+    override fun chapterListParse(response: Response): List<SChapter> = throw UnsupportedOperationException()
+
+    override fun pageListParse(response: Response): List<Page> = throw UnsupportedOperationException()
+
+    override fun imageUrlParse(response: Response): String = throw UnsupportedOperationException()
+
+    companion object {
+        private val DATE_FMT = SimpleDateFormat("EEE d MMM, yyyy", Locale.US)
```

Comment:
Since the timezone is available(?) you can add that in the format string or by setting it to PDT/PST/UTC, whichever one it uses for the date field you're reading from.

Reply:
Uhh, i don’t know the timezone, actually. All dates really just match the format string, like: »Sun 13 Jan, 2019«, no additional info. The `Locale.US` only exists to correctly handle day and month names.

Reply:
The site's footer has a timestamp with a timezone

Reply:
Huh, never noticed. Thanks, will add!

Reply:
The individual page dates have no timezone info, and i’m hesitant to just `+ " PDT"` them. However, i added timezone decoding to »dateless« archives in the next commit.